suite-native/accounts/src/selectors.ts

```diff
@@ -32,18 +35,34 @@ export const isFilterValueMatchingAccount = (account: Account, filterValue: stri
 /**
  * Filter accounts by labels, network names and included token names.
  */
-const filterAccountsByLabelAndNetworkNames = (accounts: Account[], filterValue?: string | null) => {
+const filterAccountsByLabelAndNetworkNames = (
+    accounts: readonly Account[],
+    filterValue?: string | null,
```

Comment:
Why is this undefined | null | string? In the AccountList you have the filter value defined as string.

Reply:
Good point. Originally it was possibly undefined | null. Now after refactoring olny valid input is string. a14e10331